src/components/CartContext.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+export const CartContext = React.createContext();
+
+const CartProvider = ({ children }) => {
+  const [cart, setCart] = useState([]);
+  const [orders, setOrders] = useState([]);
+
+  return (
+    <CartContext.Provider value={[cart, setCart, orders, setOrders]}>
```

Comment:
Each context must have its own responsibility... in this case, `CartContext` is in charge of updating the cart state and the orders state.

Reply:
Thanks for the comment, has already changed

Reply:
just adding to @GerryLarios comment, what you can you (for further references, not mandatory do it right now), is create a context with both properties, something like:

```JSX
 const [state, setState] = useState({ cart: [], orders: [] })
```

This way you just have one setter